libs/sdk-core/src/breez_services.rs

```diff
@@ -2645,6 +2695,39 @@ async fn get_lsp_by_id(
         .cloned())
 }
 
+/// Convenience method to get all LSPs (active and historical) relevant for registering or
+/// unregistering webhook notifications
+async fn get_notification_lsps(
+    persister: Arc<SqliteStorage>,
+    lsp_api: Arc<dyn LspAPI>,
+    node_api: Arc<dyn NodeAPI>,
+) -> SdkResult<Vec<LspInformation>> {
+    let node_pubkey = persister
+        .get_node_state()?
+        .ok_or(SdkError::generic("Node info not found"))?
+        .id;
+    let maybe_active_lsp_id = persister.get_lsp_id()?;
+    let open_peers = node_api.get_open_peers().await?;
+
+    let mut notification_lsps = vec![];
+    for lsp in lsp_api.list_used_lsps(node_pubkey).await? {
+        match matches!(maybe_active_lsp_id, Some(ref active_lsp_id) if active_lsp_id == &lsp.id) {
```

Comment:
I think we don't need to fetch the current lsp and instead we can identify it by the fee params section. If it exists then we should register for notification (because it means it can open a channel to us).

Reply:
Done: 0f0bed0eeb95f5a0612e4ff0e775d8adc9581802